config/software/berkshelf.rb

```diff
@@ -43,4 +43,20 @@
 
   gem "install pkg/berkshelf-*.gem" \
       " --no-ri --no-rdoc", env: env
+
+  if windows?
+    block "Clean up large object files" do
+      lib_path = File.join(
+        "#{install_dir.gsub(/\\/, '/')}",
+        "embedded/lib/ruby/gems/**/gems",
+        "dep-selector-libgecode-*",
+        "lib/dep-selector-libgecode/vendored-gecode/lib",
+      )
+
+      Dir.glob("#{lib_path}/*.a").each do |f|
+        puts "Deleting #{f}"
+        File.delete(f)
+      end
```

Comment:
I'd like to see this in non-windows as well, after a full build has gone through CI to make sure it's not going break the sad platforms, as Lamont mentioned.